airflow_aws_executors/batch_executor.py

```diff
@@ -118,23 +139,43 @@ def _describe_tasks(self, job_ids) -> List[BatchJob]:
             all_jobs.extend(describe_tasks_response['jobs'])
         return all_jobs
 
-    def execute_async(self, key: TaskInstanceKeyType, command: CommandType, queue=None, executor_config=None):
+    def execute_async(
+        self,
+        key: TaskInstanceKeyType,
+        command: CommandType,
+        queue: Optional[str] = None,
```

Comment:
Another problem. Suppose you declare an operator with no queue kwarg (as most operators are created). The default `queue` is based off of your [airflow cfg value for celery.default_queue](https://github.com/apache/airflow/blob/251eb7d170db3f677e0c2759a10ac1e31ac786eb/airflow/models/baseoperator.py#L370).  
That means that instead of using your `batch.job_queue` config, you'll end up using `celery.default_queue`; which is not reasonable. I see you're experimenting with `operator.default_queue`; I'm not quite familiar with this?